coding-concepts.js

```diff
@@ -16,44 +16,45 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: 10 was the output. I counted the amount of charaters and spaces in the string
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: error
+// b) Verify and explain: o was the output. i expected an error because thought [4] was only for arrays. [4] counted from 0-4 inside the string
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: Ruby was the output. index =1 therefore languages[1] was loged
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+//console.log(weekendDays.toUpperCase())
+
+// a) Your answer: ["SATURDAY", "SUNDAY"]
+// b) Verify and explain: weekendDays.toUpperCase is not a function. the weekendDays needed to have [0] or [1] to change each value to an upper case value.
```

Comment:
To expand a little further, .toUpperCase() only works on strings and not on arrays.  To achieve the result in your answer, you would need to use some type of iteration and run .toUpperCase() on each value.